.github/workflows/periodic.yml

```diff
@@ -110,6 +110,61 @@ jobs:
       docker-image: ${{ needs.linux-bionic-cuda11_6-py3_7-gcc7-debug-build.outputs.docker-image }}
       test-matrix: ${{ needs.linux-bionic-cuda11_6-py3_7-gcc7-debug-build.outputs.test-matrix }}
 
+  linux-bionic-cuda11_8-py3_8-gcc7-debug-build:
+    name: linux-bionic-cuda11.8-py3.8-gcc7-debug
+    uses: ./.github/workflows/_linux-build.yml
+    with:
+      build-environment: linux-bionic-cuda11.8-py3.8-gcc7-debug
+      docker-image-name: pytorch-linux-bionic-cuda11.8-cudnn8-py3-gcc7
+      build-with-debug: true
+      test-matrix: |
+        { include: [
+          { config: "default", shard: 1, num_shards: 4, runner: "linux.4xlarge.nvidia.gpu" },
+          { config: "default", shard: 2, num_shards: 4, runner: "linux.4xlarge.nvidia.gpu" },
+          { config: "default", shard: 3, num_shards: 4, runner: "linux.4xlarge.nvidia.gpu" },
+          { config: "default", shard: 4, num_shards: 4, runner: "linux.4xlarge.nvidia.gpu" },
+        ]}
+
+  linux-bionic-cuda11_8-py3_8-gcc7-debug-test:
+    name: linux-bionic-cuda11.8-py3.8-gcc7-debug
+    uses: ./.github/workflows/_linux-test.yml
+    needs: linux-bionic-cuda11_8-py3_8-gcc7-debug-build
+    with:
+      build-environment: linux-bionic-cuda11.8-py3.8-gcc7-debug
+      docker-image: ${{ needs.linux-bionic-cuda11_8-py3_8-gcc7-debug-build.outputs.docker-image }}
+      test-matrix: ${{ needs.linux-bionic-cuda11_8-py3_8-gcc7-debug-build.outputs.test-matrix }}
+
+  libtorch-linux-bionic-cuda11_8-py3_8-gcc7-build:
+    name: libtorch-linux-bionic-cuda11.8-py3.8-gcc7
+    uses: ./.github/workflows/_linux-build.yml
+    with:
+      build-environment: libtorch-linux-bionic-cuda11.8-py3.8-gcc7
+      docker-image-name: pytorch-linux-bionic-cuda11.8-cudnn8-py3-gcc7
+      build-generates-artifacts: false
+
+  win-vs2019-cuda11_8-py3-build:
+    name: win-vs2019-cuda11.8-py3
+    uses: ./.github/workflows/_win-build.yml
+    with:
+      build-environment: win-vs2019-cuda11.8-py3
+      cuda-version: "11.8"
+      test-matrix: |
+        { include: [
+          { config: "default", shard: 1, num_shards: 3, runner: "windows.g5.4xlarge.nvidia.gpu" },
+          { config: "default", shard: 2, num_shards: 3, runner: "windows.g5.4xlarge.nvidia.gpu" },
+          { config: "default", shard: 3, num_shards: 3, runner: "windows.g5.4xlarge.nvidia.gpu" },
+          { config: "force_on_cpu", shard: 1, num_shards: 1, runner: "windows.4xlarge" },
+        ]}
```

Comment:
All three windows tests hards are failing, with errors like the following. Can you please take a look? I think we'll need to disable these tests for now since they're failing consistently on periodic

```
2023-01-24T03:25:03.2580503Z ERROR (0.006s)
2023-01-24T03:25:03.2580979Z   test_multihead_attention_dtype_batch_first_cuda_float16 (__main__.TestMultiheadAttentionNNDeviceTypeCUDA) ...     test_multihead_attention_dtype_batch_first_cuda_float16 errored - num_retries_left: 2
2023-01-24T03:25:03.2581378Z Traceback (most recent call last):
2023-01-24T03:25:03.2581890Z   File "C:\actions-runner\_work\pytorch\pytorch\build\win_tmp\build\torch\testing\_internal\common_device_type.py", line 414, in instantiated_test
2023-01-24T03:25:03.2582234Z     raise rte
2023-01-24T03:25:03.2582586Z   File "C:\actions-runner\_work\pytorch\pytorch\build\win_tmp\build\torch\testing\_internal\common_device_type.py", line 401, in instantiated_test
2023-01-24T03:25:03.2582948Z     result = test(self, **param_kwargs)
2023-01-24T03:25:03.2583378Z   File "C:\actions-runner\_work\pytorch\pytorch\build\win_tmp\build\torch\testing\_internal\common_device_type.py", line 1010, in only_fn
2023-01-24T03:25:03.2583719Z     return fn(slf, *args, **kwargs)
2023-01-24T03:25:03.2584077Z   File "C:\actions-runner\_work\pytorch\pytorch\test\nn\test_multihead_attention.py", line 640, in test_multihead_attention_dtype_batch_first
2023-01-24T03:25:03.2584496Z     model = nn.MultiheadAttention(embed_dim, num_heads, batch_first=True).cuda().to(dtype)
2023-01-24T03:25:03.2584961Z   File "C:\actions-runner\_work\pytorch\pytorch\build\win_tmp\build\torch\nn\modules\module.py", line 1132, in to
2023-01-24T03:25:03.2585289Z     return self._apply(convert)
2023-01-24T03:25:03.2585661Z   File "C:\actions-runner\_work\pytorch\pytorch\build\win_tmp\build\torch\nn\modules\module.py", line 784, in _apply
2023-01-24T03:25:03.2586057Z     module._apply(fn)
2023-01-24T03:25:03.2586468Z   File "C:\actions-runner\_work\pytorch\pytorch\build\win_tmp\build\torch\nn\modules\module.py", line 807, in _apply
2023-01-24T03:25:03.2586789Z     param_applied = fn(param)
2023-01-24T03:25:03.2587196Z   File "C:\actions-runner\_work\pytorch\pytorch\build\win_tmp\build\torch\nn\modules\module.py", line 1130, in convert
2023-01-24T03:25:03.2587602Z     return t.to(device, dtype if t.is_floating_point() or t.is_complex() else None, non_blocking)
2023-01-24T03:25:03.2587932Z RuntimeError: CUDA error: no kernel image is available for execution on the device
2023-01-24T03:25:03.2588367Z CUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.
2023-01-24T03:25:03.2588716Z For debugging consider passing CUDA_LAUNCH_BLOCKING=1.
2023-01-24T03:25:03.2589013Z Compile with `TORCH_USE_CUDA_DSA` to enable device-side assertions.
```

https://hud.pytorch.org/pytorch/pytorch/commit/d8aa68c683bdf31f237bffb734b6038bc4f63898

Reply:
cc: @ptrblck  @atalman

Reply:
@ZainRizvi looking into this issue, was there AMI update recently ? I don't see any failures on https://github.com/pytorch/pytorch/actions/runs/3961152467/jobs/6788616724